### PR TITLE
xz 5.6.0

### PR DESCRIPTION
**Pull request by miccal**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/x/xz.rb [hunk 1]
```diff
@@ -4,11 +4,11 @@ class Xz < Formula
...
-  mirror "https://archive.org/download/xz-5.4.6/xz-5.4.6.tar.gz"
-  mirror "http://archive.org/download/xz-5.4.6/xz-5.4.6.tar.gz"
-  sha256 "aeba3e03bf8140ddedf62a0a367158340520f6b384f75ca6045ccc6c0d43fd5c"
+  url "https://github.com/tukaani-project/xz/releases/download/v5.6.0/xz-5.6.0.tar.gz"
+  mirror "https://downloads.sourceforge.net/project/lzmautils/xz-5.6.0.tar.gz"
+  mirror "https://archive.org/download/xz-5.6.0/xz-5.6.0.tar.gz"
+  mirror "http://archive.org/download/xz-5.6.0/xz-5.6.0.tar.gz"
+  sha256 "0f5c81f14171b74fcc9777d302304d964e63ffc2d7b634ef023a7249d9b5d875"
   license all_of: [
     :public_domain,
```
Comment (Bo98) [inline Formula/x/xz.rb:13]:
```suggestion
    "0BSD",
```

changed as of this release

Comment (miccal) [reply Formula/x/xz.rb]: Thank you @Bo98.

**Comment by cho-m** [comment]
archive.org URLs need to be created before we can run this.

**Comment by p-linnane** [comment]
> archive.org URLs need to be created before we can run this.

Done.

**Comment by miccal** [comment]
> > archive.org URLs need to be created before we can run this.
> 
> Done.

Thank you @p-linnane.

**Review by p-linnane — APPROVED** [review]
Skipping recursive dependents makes this no longer need to be a long build.

**Pull request merged**